### PR TITLE
CLOUDP-265391 Move Local Deployment RHEL tests to be in a periodic build

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Move Local Deployment RHEL tests to be in a periodic build

_Jira ticket:_ CLOUDP-265391

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by jeroenvervaeke — DISMISSED** [review]

File: build/ci/periodic_build.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+exec_timeout_secs: 4200 # automatically fail any task if it takes longer than 1h10m to finish.
+include:
+  - filename: build/ci/evergreen_common.yml
+  
+variables:
+  - &go_linux_version
+      go_root: "/opt/golang/go1.22"
+      go_bin: "/opt/golang/go1.22/bin"
+      go_base_path: ""
```
Comment (tibulca) [inline build/ci/periodic_build.yml:23]: question: can we have these vars in common? I see them in build/ci/evergreen.yml too

File: build/ci/evergreen_common.yml [hunk 2]
```diff
@@ -0,0 +1,199 @@
+# Copyright 2021 MongoDB Inc
```
Comment (tibulca) [inline build/ci/evergreen_common.yml:1]:
```suggestion
# Copyright 2024 MongoDB Inc
```

File: build/ci/periodic_build.yml (hunk 1 above)
Comment (fmenezes) [reply build/ci/periodic_build.yml:23]: unfortunately not the yaml anchor is per file

File: build/ci/evergreen.yml [hunk 3]
```diff
@@ -16,6 +16,9 @@ exec_timeout_secs: 4200 # automatically fail any task if it takes longer than 1h
 stepback: true
 command_type: system
 pre_error_fails_task: true
+include:
+  - filename: build/ci/release.yml
+  - filename: build/ci/evergreen_common.yml
 
 variables:
   - &go_linux_version
```
Comment (tibulca) [inline build/ci/evergreen.yml:24]: some are already defined in common ( e.g. &go_env), can we remove them from here?

Comment (fmenezes) [reply build/ci/evergreen.yml:24]: as mentioned earlier we have to have them on both files

Comment (fmenezes) [reply build/ci/evergreen.yml:24]: same for release.yml

**Review by tibulca — APPROVED** [review]

**Pull request merged**